projects/cashmere/src/lib/app-switcher/app-switcher.component.html

```diff
@@ -12,9 +12,9 @@ <h4>Applications
     <ng-container *ngIf="applications?.length > 0">
         <div class="apps">
             <div class="app" *ngFor="let app of applications">
-                <a class="thumbnail" [title]="app.Description" [href]="app.ServiceUrl" target="_blank">
-                    <img class="thumbnail-img" [height]="iconHeight" [src]="app.Icon" [ngStyle]="{'max-width': iconHeight + 'px'}">
```

Comment:
Hey @jacquie - looks like the `ngStyle` block I added for max-width got lost in the merge.  That's important because it keeps the icons from overflowing their square.

Reply:
Oh no - sorry!  I'll add it back in...

Reply:
hooray!  thank you :-)